adserver/tests/test_decision_engine.py

```diff
@@ -230,23 +230,23 @@ def test_flight_geo_targeting(self):
         ad, _ = self.backend.get_ad_and_placement()
         self.assertTrue(ad in (self.advertisement1, self.advertisement2))
 
-        self.backend.country_code = "US"
+        self.backend.geolocation = GeolocationData("US")
         ad, _ = self.backend.get_ad_and_placement()
         self.assertEqual(ad, self.advertisement1)
 
-        self.backend.country_code = "MX"
+        self.backend.geolocation = GeolocationData("MX")
         ad, _ = self.backend.get_ad_and_placement()
         self.assertTrue(ad in (self.advertisement1, self.advertisement2))
 
-        self.backend.country_code = "FO"
+        self.backend.geolocation = GeolocationData("FO")
         ad, _ = self.backend.get_ad_and_placement()
         self.assertEqual(ad, self.advertisement2)
 
-        self.backend.country_code = "AZ"
+        self.backend.geolocation = GeolocationData("AZ")
         ad, _ = self.backend.get_ad_and_placement()
         self.assertIsNone(ad)
 
-        self.backend.country_code = "RANDOM"
+        self.backend.geolocation = GeolocationData("T1")
         ad, _ = self.backend.get_ad_and_placement()
         self.assertEqual(ad, self.advertisement2)
```

Comment:
So we'll serve an ad to Tor users if a flight has only exclude countries, but nothing included?

Reply:
That's correct as written. Maybe add it to the exclusion list?